docs/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Version 3.34.0-eisop2 (May ?, 2023)
 
 **User-visible changes:**
 
+The Initialization Checker supports new qualifier:
```

Comment:
Please make this a complete sentence that motivates why this is useful. Also, you should add a paragraph about the new qualifier to the manual.

Reply:
Already added.